src/subscript/params2csv/params2csv.py

```diff
@@ -103,16 +180,13 @@ def main() -> None:
         )  # if key is repeated, keep the last one.
         transposed = paramtable.set_index("key").transpose()
         if args.filenamecolumnname in transposed.columns:
-            print(
-                "Column name "
-                + args.filenamecolumnname
-                + " was already in "
-                + parameterfilename
-                + ", not writing"
-            )
-            print(
-                "this filename into CSV output. Use --filenamecolumnname to avoid this."
+            logger.info(
+                "Column name %s was already in %s, not writing this filename "
+                "into CSV output. Use --filenamecolumnname to avoid this.",
+                args.filenamecolumnname,
+                parameterfilename,
             )
+            logger.info()
```

Comment:
This empty one could probably be removed entirely now as well 😄

Reply:
of course :rofl: